wandb/sdk/data_types/saved_model.py

```diff
@@ -80,12 +83,12 @@ class _SavedModel(WBValue, Generic[SavedModelObjType]):
     _path_extension: ClassVar[str]
 
     _model_obj: Optional["SavedModelObjType"]
-    _path: Optional[str]
-    _input_obj_or_path: Union[SavedModelObjType, str]
+    _path: Optional[Path]
```

Comment:
is it now only `Path`?

Reply:
This is internal, so it can safely be guaranteed to be a `pathlib.Path`